### PR TITLE
fixed bootstrap btn styling

### PR DESCRIPTION
**Pull request by bam95899**
Fixes issue #359

### CODE REVIEW

**Comment by bam95899** [comment]
There is not a lot to this PR. The majority of buttons already had `btn-<name>` classes applied to them, but the `btn` styling had !important applied to the `background-color` attribute so all buttons were forced to be blue.

File: tinker/static/assets/css/tinker.css [hunk 1]
```diff
@@ -175,6 +180,11 @@ a#remove_new_job_button {
     border-color: #f04124 !important;
 }
 
+.danger {
```
Comment (c-schwarze) [inline tinker/static/assets/css/tinker.css]: don't we already have .btn-danger with bootstrap? https://getbootstrap.com/docs/4.3/components/buttons/

File: tinker/static/assets/css/tinker.css [hunk 2]
```diff
@@ -175,6 +180,11 @@ a#remove_new_job_button {
     border-color: #f04124 !important;
 }
 
+.btn-danger {
+    background-color: #f04124 !important;
+    border-color: #f04124 !important;
+}
```
Comment (c-schwarze) [inline tinker/static/assets/css/tinker.css]: Do we need a custom `btn-danger` class? or can we get by with the default? https://getbootstrap.com/docs/4.3/components/buttons/

Comment (bam95899) [reply tinker/static/assets/css/tinker.css]: Sort of.

With the custom btn-danger class our button looks like this:
image

Without the class the button looks like this:
image

And without any of the CSS that changes the button blue, it looks like this, even though the btn-danger class is applied to the button and the btn-outline-danger class is not:
image

Comment (c-schwarze) [reply tinker/static/assets/css/tinker.css]: Gotcha. Lets go ahead and push this through

**Comment by c-schwarze** [comment]
@bam95899 when you have a chance, can you merge this, test on xp, and make sure it works on prod?

**Pull request merged**